app/components/ClassScreen.tsx

```diff
@@ -0,0 +1,260 @@
+            let imageFileName = `${getCurrentDateTimeString()}.${imageExtension}`;
+            uploadImageHandler(pickerResult.assets[0].uri, imageFileName);
+        }
+        setAttachPopupIsVisible(false);
+    };
+
+    const handleCameraPress = async () => {
+        let permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+        if (permissionResult.granted === false) {
+            alert("Permission to access camera is required!");
+            return;
+        }
+      
+        const pickerResult = await ImagePicker.launchCameraAsync({ allowsEditing: true, quality: 1 });
+        if (!pickerResult.canceled) {
+            const imageExtension = pickerResult.assets[0].mimeType?.split("/")[1];
+            let imageFileName = `${getCurrentDateTimeString()}.${imageExtension}`;
+            uploadImageHandler(pickerResult.assets[0].uri, imageFileName);
+        }
+        setAttachPopupIsVisible(false);
+    };
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.chatContainer}>
+                <ScrollView style={styles.chatContent}>
+
+                </ScrollView>
+                <View style={styles.sendMessageContainer}>
+                    <View style={styles.chatInputContainer}>
+                        <TextInput
+                            style={styles.chatInputField}
+                            placeholder="Nachricht schreiben..."
+                            multiline={true}
+                            placeholderTextColor={Colors.secondary}
+                        />
+                        <View style={styles.attachButtons}>
+
+                            <Popover
+                                isVisible={attachPopupIsVisible}
+                                onRequestClose={() => setAttachPopupIsVisible(false)}
+                                animationConfig={{
+                                    duration: 300
+                                }}
+                                from={(
+                                    <TouchableOpacity style={styles.attachButton} onPress={() => setAttachPopupIsVisible(true)}>
+                                        <Ionicons name="attach" size={30} color={Colors.secondary} />
+                                    </TouchableOpacity>
+                                )}>
+                                <View style={styles.settingModal}>
+                                    <TouchableOpacity style={styles.settingButton} onPress={() => handleImagePick()}>
+                                        <Feather name="image" size={25} color={Colors.primary} />
+                                        <Text style={styles.settingText}>BILD</Text>
+                                    </TouchableOpacity>
+                                    <TouchableOpacity style={styles.settingButton} onPress={() => uploadFileHandler()}>
+                                        <Feather name="file-text" size={25} color={Colors.primary} />
```

Comment:
For iPhone it looks like this:
 <img src="https://github.com/lern-fair-file-sharing/app/assets/68401216/3931f44c-0055-486c-a0d1-28f38d1cc996" alt="Cute snap of my gorgeous view" width="300">

Reply:
Okay, just like on my iPad, oh man...